code/game/objects/items/weapons/dna_injector.dm

```diff
@@ -1190,3 +1190,35 @@
 	New()
 		block = REMOTETALKBLOCK
 		..()
+
+/obj/item/weapon/dnainjector/nofail/randompower/New(newloc)
+    var/type = pick(/obj/item/weapon/dnainjector/nofail/hulkmut,
```

Comment:
I got the code from pjb and i have fucking no idea how it works don't look at me.